_examples/simple/simple.go

```diff
@@ -12,3 +12,10 @@ func Func() {}
 func Add(i, j int) int {
 	return i + j
 }
+
+func Bool(b bool) bool {
```

Comment:
it's a bit of a confusing test function.
why not just:

```go
func Bool(b bool) bool {
   return b
}
```

or perhaps:
```go
func Equal(a, b int) bool {
   return a == b
}
```

(or did you want to also test `bool`eans as arguments of functions? in that case I would stick with my former `Bool()` function)